app/boards/shields/pancake/pancake.zmk.yml

```
@@ -0,0 +1,8 @@
+file_format: "1"
+id: pancake
+name: Pancake
+type: shield
+url: https://mkultra.click/pancake-keyboard-kit
+required: [pro_micro]
```

Typo, this should be `requires`:

```suggestion
requires: [pro_micro]
```

I believe that's causing the docs build failures.

---

my bad, good catch.  I've re-read the docs on testing documentation so I'll know how to catch this before hand.